source/MissionPanel.cpp

```diff
@@ -745,6 +746,14 @@ void MissionPanel::Accept()
 	int crewToFire = 0;
 	if(toAccept.Passengers())
 		crewToFire = toAccept.Passengers() - player.Cargo().BunksFree();
+	// Automatically fire crew, if appropriate.
+	if(crewToFire > 0 && Preferences::Has("Auto fire crew to make space"))
```

Comment:
If both cargoToSell and crewToFire are non-zero, then it is possible that crew automatically gets fired, while the player chooses not to sell the cargo.